sats-dna/src/main/kotlin/com/sats/dna/components/button/SatsButton.kt

```diff
@@ -103,10 +104,19 @@ private sealed interface IconContent {
 }
 
 @Composable
-private fun buttonPadding(isLarge: Boolean) = PaddingValues(
-    horizontal = SatsTheme.spacing.m,
-    vertical = if (isLarge) SatsTheme.spacing.m else SatsTheme.spacing.xs,
-)
+private fun buttonPadding(isLarge: Boolean): PaddingValues {
+    val horizontal = animateDpAsState(
```

Comment:
Does this have any effect when the value is always `SatsTheme.spacing.m`?

Reply:
Fair. :rofl: